.travis.yml

```diff
@@ -34,7 +34,6 @@ matrix:
       jdk: oraclejdk8
       env: "JRUBY_OPTS='-Xcompile.invokedynamic=true --debug'"
 before_install:
-  - gem update --system
```

Comment:
if you touch this file, might as well make the whole thing work with latest rubygems and bundler